cadc-storage-adapter-fs/src/test/java/org/opencadc/inventory/storage/fs/FileSystemStorageAdapterTest.java

```diff
@@ -111,7 +112,7 @@
     private static final byte[] data = dataString.getBytes();
 
     static {
-        Log4jInit.setLevel("org.opencadc.inventory", Level.INFO);
+        Log4jInit.setLevel("org.opencadc.inventory", Level.DEBUG);
```

Comment:
Prefer to have these at INFO in master; devs turn it up to DEBUG if they need more details than INFO provides

Reply:
done